src/cljs/doit/events.cljs

```diff
@@ -21,6 +21,33 @@
      :after (fn [context]
               (check-db-spec context (get-in context [:effects :db]))))))
 
+(defn token-headers-map [cofx]
```

Comment:
Shouldn't this be in the `auth` namespace?

Reply:
Moving this to `auth` introduces circular dependency as `auth` dispatches events from `events`

Reply:
Alright, this is ok for now.